diff --git a/node_modules/class-transformer/TransformOperationExecutor.js b/node_modules/class-transformer/TransformOperationExecutor.js
index dec36c7..d69f416 100644
--- a/node_modules/class-transformer/TransformOperationExecutor.js
+++ b/node_modules/class-transformer/TransformOperationExecutor.js
@@ -252,7 +252,7 @@ class TransformOperationExecutor {
                     if ((this.transformationType === enums_1.TransformationType.PLAIN_TO_CLASS ||
                         this.transformationType === enums_1.TransformationType.CLASS_TO_CLASS) &&
                         // eslint-disable-next-line @typescript-eslint/unbound-method
-                        ((descriptor && !descriptor.set) || newValue[newValueKey] instanceof Function))
+                        ((descriptor && !descriptor.set && !('value' in descriptor)) || newValue[newValueKey] instanceof Function))
                         //  || TransformationType === TransformationType.CLASS_TO_CLASS
                         continue;
                 }
